galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
It's fine as long as it works correctly. I won't personally implement things that I can't find documented anywhere in the wiki.